addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComSecurity2MessageTest.java

```diff
@@ -8,20 +8,38 @@
  */
 package org.openhab.binding.rfxcom.internal.messages;
 
+import static org.junit.Assert.assertEquals;
 import static org.openhab.binding.rfxcom.internal.messages.RFXComBaseMessage.PacketType.SECURITY2;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComSecurity2Message.SubType;
+
+import javax.xml.bind.DatatypeConverter;
 
 import org.junit.Test;
-import org.openhab.binding.rfxcom.internal.exceptions.RFXComMessageNotImplementedException;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
 
 /**
  * Test for RFXCom-binding
  *
  * @author Martin van Wingerden
+ * @author Mike Jagdis
```

Comment:
Maybe change mine to initial contribution of empty test and add after your name that you added the real test